lib/util/mp4_box_parsers.js

```diff
@@ -252,13 +259,16 @@ shaka.util.ParsedTREXBox;
 /**
  * @typedef {{
  *    sampleCount: number,
- *    sampleData: !Array.<shaka.util.ParsedTRUNSample>
+ *    sampleData: !Array.<shaka.util.ParsedTRUNSample>,
+ *    dataOffset: ?number
  * }}
  *
  * @property {number} sampleCount
  *   As per the spec: the number of samples being added in this run;
  * @property {!Array.<shaka.util.ParsedTRUNSample>} sampleData
  *   An array of size <sampleCount> containing data for each sample
+ * @property {number} fdataOffset
```

Comment:
typo: fdataOffset should say dataOffset, and number should say ?number.

Not a compiler error because the `@property` tags are for jsdoc only.

Reply:
Done